### ISSUE TITLE
Search tool tip

### ISSUE THREAD

**Issue by haesleinhuepf** [NEUBIAS/bise#193]
Hi @PerrineGilloteaux ,

I just noticed this tool-tip "Look also to Advanced Search", which is not very helpful as the link to the advanced search is just below:

![image](https://user-images.githubusercontent.com/12660498/235288123-c09b826e-cf4b-40aa-b822-1647f586a4c2.png)

I propose to change the tooltip to something like:
`Enter search terms such as 'Cell tracking' together with 'Brightfield microscopy' to narrow down the results to your needs.`

**Comment by PerrineGilloteaux** [comment]
Hi @haesleinhuepf  related to https://github.com/NEUBIAS/bise/issues/195

I would suggest the following tooltip for the state of search: here (your screen capture) "Enter search term that will  looked in the full description of the tool (soft/dataset/training material). Advanced Search can be used for a search based on structured vocabularies". 

What do you think?

**Comment by PerrineGilloteaux** [comment]
Also maybe even more useful: mentionning that "space" denotes “+” search meaning OR and will return more results as it will list all tools containing at least one of the listed terms. Adding “&” in the search means AND, and will return 
tools matching the list of terms. so here is the new proposal:

 **Enter search term that will looked in the full description of the tool (soft/dataset/training material). example: "Cell Tracking" will return entries containing Cell OR Tracking, "Cell&Tracking" will return entries containing Cell AND Tracking.
 Advanced Search can be used for a search based on structured vocabularies**

**Comment by haesleinhuepf** [comment]
> Also maybe even more useful: mentionning that "space" denotes “+” search meaning OR and will return more results as it will list all tools containing at least one of the listed terms. Adding “&” in the search means AND, and will return
> tools matching the list of terms. so here is the new proposal:

@PerrineGilloteaux  Would it be possible to change the default of <space> to AND? When I enter "Tracking Fiji" I certainly do not want all Fiji tools to be listed, but only the tracking tools...

**Comment by PerrineGilloteaux** [comment]
yes. (but note that some people had different views at the taggathons, asking that it should be OR , that's why we set up it like this). I am waiting for Kota's version upgrade, once done I can commit the change that will happen (note for my self) in /admin/structure/views/view/testsearch Configure filter criterion: Search: Fulltext search contains all of these words instead of any of these words
